src/utils/tasksUtils.ts

```diff
@@ -83,6 +84,26 @@ function isQuarkusDevTask(task: Task, projectBuildSupport: BuildSupport): boolea
       || shellExecution.commandLine.includes(projectBuildSupport.getWrapperWindows()));
 }
 
+export async function shouldUpdateTaskCommand(folderPath: string, task: Task, projectBuildSupport: BuildSupport): Promise<boolean> {
+
+  const execution: ProcessExecution | ShellExecution | CustomExecution = task.execution;
+
+  if (!execution || !(execution as ShellExecution)) {
+    return false;
+  }
+
+  const shellExecution: ShellExecution = execution as ShellExecution;
+
+  if (shellExecution
```

Comment:
Should be able to simplify this a bit with https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html .

Also I think instead of `if (array.length > 0)` you can simply use `if(array)`